go/vt/mysqlctl/replication.go

```diff
@@ -69,8 +74,8 @@ func changeMasterArgs(params *sqldb.ConnParams, masterHost string, masterPort in
 }
 
 // parseSlaveStatus parses the common fields of SHOW SLAVE STATUS.
-func parseSlaveStatus(fields map[string]string) *proto.ReplicationStatus {
-	status := &proto.ReplicationStatus{
+func parseSlaveStatus(fields map[string]string) proto.ReplicationStatus {
```

Comment:
Then you meet with Anthony and you guys figure it out, he's the one who made me change proto.ReplicationPosition from pointer to value, as it only contains a single interface object (which is already a pointer, basically)...

Come to think of it, I don't mind one being a value and the other one a pointer. I can do that too. 

@enisoc @aaijazi what do you say?